packages/module/src/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+import { CloseIcon } from '@patternfly/react-icons';
+import clsx from 'clsx';
+import { createUseStyles } from 'react-jss';
+
+const useStyles = createUseStyles({
+  '& float-right': {
+    float: 'right'
+  },
+  '&  no-padding': {
+    padding: 0
+  }
+});
+
+export interface CloseButtonProps {
+  /** Additional styling to apply to the close button. */
+  additionalClassName?: string;
```

Comment:
I would rename this prop for consistency - we could also start some migration notes for OpenShift

```suggestion
  /** Additional styling to apply to the close button. */
  className?: string;
```

Reply:
also can be removed when we extend Button props